src/platform/muen/componolit-interfaces-muchannel_reader.adb

```diff
@@ -0,0 +1,53 @@
+
+with System;
+
+package body Componolit.Interfaces.Muchannel_Reader with
+   SPARK_Mode
+is
```

Comment:
What's the reason for this functions to have SPARK mode off? Isn't Muchannel in SPARK?